bluemarket/src/index.js

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import App from './App.jsx';
 import * as serviceWorker from './serviceWorker';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Where is this coming from to clarify, is this something coming from React

Reply:
This import goes into the node_modules folder. NPM installed a default bootstrap.css file of that name in our project. These can be referenced as if your in the same folder for cleanliness (thats how they do it online).